javascript/JS-Array-Methods/js-array-methods_filter/index.js

```diff
@@ -23,14 +23,18 @@ const cards = [
   },
 ];
 
-const onlyCardWithIdTwo = null;
+const onlyCardWithIdTwo = cards.filter((card) => card.id == 2);
```

Comment:
Given id: "2", is a string and here you are searching for number `cards.filter((card) => card.id == 2);` .It should then be
const onlyCardWithIdTwo = cards.filter((card) => card.id == "2");